### DISCUSSION TITLE
How can I get global coordinates on OSM

### DISCUSSION THREAD

**Discussion by fakca** [foxglove/community#730] [Q&A] [answered]
I am experiencing an issue with Foxglove while trying to obtain global coordinates by clicking on the OpenStreetMap. Clicking on the map does not provide me with the latitude and longitude values corresponding to the selected location.
Is there such a feature on FoxGlove Studio? If yes, how can I get WGS84 coordinates on OSM with Foxglove Studio?
(I aim to publish these global coordinates to a ROS topic)

Thank you.

**Comment by defunctzombie** [comment, marked as answer]
The map panel does not have a feature for clicking and getting the coordinates; it only offers display of your existing data. If you want to lookup the coordinates for one of your messages then you could use the Raw Message panel to inspect the message.
